cleverhans/feat_advs.py

```diff
@@ -0,0 +1,290 @@
+import numpy as np
```

Comment:
Module level docstring

Reply:
You should put a link to the paper in here somewhere, or I think most people won't know what this is.

Also, for your sake, you probably want to give people instructions for citing the paper :)

Reply:
Docstring added in 4ec1f53. Regarding citation, is it enough to include the link to arxiv?

Reply:
Yes it should be fine to add the link, at least this is what we have done so far with other attacks :)